index.html

```diff
@@ -69,7 +69,7 @@
 
                 <div class="outside-in-headerTitle">
                         <span class="outside-in-headerTitle-icon"></span>
-                        <h1>OUTSIDE·IN</h1>
+                        <h1>OUTSIDE•IN</h1>
```

Comment:
Another option here, would be to use a named HTML entity `&bull;`

See http://www.w3.org/wiki/Common_HTML_entities_used_for_typography